core/src/main/java/com/novoda/landingstrip/LandingStrip.java

```diff
@@ -185,8 +200,11 @@ private void forceDrawIndicatorAtPosition(int position) {
         invalidate();
     }
 
+    public void detach(ViewPager viewPager) {
```

Comment:
I spotted the `attach` method! 👍  good to have the matching `detach` ignore Q's 1 & 3